h/services/list_groups.py

```diff
@@ -77,6 +77,24 @@ def session_groups(self, authority, user=None):
 
         return world_group + user_groups
 
+    def associated_groups(self, user=None):
+        """
+        Return a list of groups relevant to this session/profile (i.e. user).
```

Comment:
I think it would be clearer here just to say "Return a list of groups relevant to a user". "session/profile" might imply we are talking the current request.

The comment below could then be reworded to explain that a user is associated with a group if they are a creator or member of it.